fastlane/lanes/build.rb

```diff
@@ -85,13 +85,20 @@
       path.include?(options[:name])
     end.first
 
-    UI.user_error!("Unable to find .xctestrun file at #{build_products_path}") if xctestrun_path.nil? || !File.exist?((xctestrun_path))
+    UI.user_error!("Unable to find .xctestrun file at #{build_products_path}.") if xctestrun_path.nil? || !File.exist?((xctestrun_path))
 
-    inject_buildkite_analytics_environment(xctestrun_path: xctestrun_path) if buildkite_ci?
+    # Temporary skip Buildkite Analytics for tests running on Jetpack builds as it's not setup yet
+    # To be added in: https://github.com/wordpress-mobile/WordPress-iOS/issues/20336
```

Comment:
We should be able to remove this now. See my comment in https://github.com/wordpress-mobile/WordPress-iOS/issues/20336#issuecomment-1471262770

Reply:
removed the condition in https://github.com/wordpress-mobile/WordPress-iOS/pull/20301/commits/2d0f77ea9cb3dff630b5f2af7f52c6d5ca8fe823

Jetpack tests are in Buildkite Analytics now 🎉 
[Jetpack UI Tests - iPhone](https://buildkite.com/organizations/automattic/analytics/suites/jetpack-ios-ui-tests-iphone/runs/1bcdffc5-95e4-46e6-8398-333041293868) 
[Jetpack UI Tests - iPad](https://buildkite.com/organizations/automattic/analytics/suites/jetpack-ios-ui-tests-ipad/runs/f4a394d3-245c-42c6-9f32-e95d988da5e0)

thanks for making the update @mokagio!